src/pages/contact/styles.module.css

```diff
@@ -0,0 +1,88 @@
+.myform {
```

Comment:
use grid for layout for good responsive design

Reply:
Because something is not working 

![image](https://github.com/CodeHarborHub/codeharborhub/assets/99037494/8a2abb25-fbd2-4ac2-abdf-252461d42215)